sdk/search/Azure.Search.Documents/apis.txt

```diff
@@ -9,7 +9,7 @@ namespace Azure.Search.Documents
     public class IndexDocumentsOptions { }
     public class SearchFilter { }
 
-    public class SearchIndexClient
+    public class SearchClient
```

Comment:
`SearchClient`  -> `SearchDocumentClient`? `SearchClient` sounds like top/common one for `SearchIndexClient` ,etc.

Reply:
>  sounds like top/common one

That's actually the idea.  Since 98% of users want to start with searching, Krzysztof thinks we should give that the best name.